plugin/sampling/strategystore/static/strategy.go

```diff
@@ -17,8 +17,9 @@ package static
 // strategy defines a sampling strategy. Type can be "probabilistic" or "ratelimiting"
 // and Param will represent "sampling probability" and "max traces per second" respectively.
 type strategy struct {
-	Type  string  `json:"type"`
-	Param float64 `json:"param"`
+	Type                string               `json:"type"`
+	Param               float64              `json:"param"`
+	OperationStrategies []*operationStrategy `json:"operation_strategies"`
```

Comment:
This is not a good encapsulation, a strategy is a "simple" struct. If necessary, the overall config can have a separate service-less per-operation section, but we should not break the notion of a simple strategy.